6MI0800226/week2.cpp

```diff
@@ -0,0 +1,160 @@
+#include <iostream>
+#include <fstream>
+#include <assert.h>
+ 
+const int BUFF_SIZE = 1024;
+void saveResultsInFile(int num1, int num2, const char location[BUFF_SIZE])
+{
+    std::ofstream file(location);
+ 
+    if (!file.is_open())
+        return;
```

Comment:
Probably returning / throwing something to indicate an error is a good idea. Currently you'll never know what happened - information got saved or file failed to open :thinking: